tlsutil/generate.go

```diff
@@ -62,23 +106,23 @@ func GenerateCA(signer crypto.Signer, sn *big.Int, days int, constraints []strin
 		SubjectKeyId:          id,
 	}
 
-	if len(constraints) > 0 {
+	if len(opts.Constraints) > 0 {
 		template.PermittedDNSDomainsCritical = true
-		template.PermittedDNSDomains = constraints
+		template.PermittedDNSDomains = opts.Constraints
```

Comment:
Should constraints here be named something like `DNSNames` or similar? Seems like that would be more descriptive of how the opt is used.

Reply:
I opted for `PermittedDNSDomains` because why not...